c2corg_api/views/document.py

```diff
@@ -186,6 +191,11 @@ def _create_document(
         document = schema.objectify(document_in)
         document.document_id = None
 
+        if document.geometry:
```

Comment:
My question was more about `validate_geometry_type`. Why do you still need to call this function?